agent/api/task/task_linux_test.go

```diff
@@ -67,6 +70,16 @@ const (
 	testExecutionCredentialsID = "testExecutionCredentialsID"
 
 	defaultCPUPeriod = 100 * time.Millisecond // 100ms
+
+	scContainerName      = "service-connect"
+	scEgressListenerPort = 12345
+	scInterceptPort      = 8080
+	scListenerPort       = 15000
+	scPauseIPv4          = "172.0.0.2"
+)
+
+var (
+	scPauseContainerName = fmt.Sprintf("%s-%s", NetworkPauseContainerName, scContainerName)
```

Comment:
nit. feel that we use this in more than one place. Should we turn this into a const template in task.go and reuse it in following lines?
* https://github.com/aws/amazon-ecs-agent/blob/feature/service-connect/agent/api/task/task.go#L1504
* https://github.com/aws/amazon-ecs-agent/blob/feature/service-connect/agent/api/task/task.go#L1534

Reply:
Will do